client/src/components/Badge.js

```diff
@@ -1,4 +1,4 @@
-import React from "react";
+import React, { useState } from "react";
```

Comment:
You don't have to import React here

Reply:
we still have the es-lint rule for having react imported ... maybe we'll change that in a future PR for the whole project and disable `'React' must be in scope when using JSXeslintreact/react-in-jsx-scope
`